src/redux/reducers/cats.js

```diff
@@ -18,6 +19,12 @@ export default function cats(state = initialState, action) {
   switch (action.type) {
     case 'cats/RANDOM_CAT':
       return { ...state, cat: randomCat() }
+
+    case 'cats/ADOPT_CAT':
+      return {
+        ...state,
+        adoptedCats: state.adoptedCats.concat(action.ADOPT_CAT),
```

Comment:
This is looking good, you are adding a value into the adopted cats array. 

The issue with this code is the value you are adding into the array, eg. `action.ADOPT_CAT`

What you are saying in this line is get the `ADOPT_CAT` property of the `action` object. If you look at [this line](https://github.com/hollymrogers/redux-task/pull/3/files#diff-414e8eea84b2897ade73e19f5b8a94b0R2) however, we can see that our action object doesn't have a `ADOPT_CAT` property, it only has `type`.

So at the moment we will be adding `null` into the array (which you may be able to see in your console)